src/test/java/com/g4s8/ghman/data/DatabaseExtension.java

```diff
@@ -51,18 +48,14 @@
 
     @Override
     public void beforeAll(final ExtensionContext context) throws Exception {
-        final SimpleDataSource dsrc = new SimpleDataSource();
-        dsrc.setUrl(this.postgres.start());
-        DatabaseExtension.src = dsrc;
-        final Flyway flyway = new Flyway();
-        flyway.setDataSource(DatabaseExtension.src);
-        flyway.migrate();
+        this.src.setUrl(this.postgres.start());
+        Flyway.configure().dataSource(this.src).load().migrate();
     }
 
     @Override
     public void afterAll(final ExtensionContext context) throws Exception {
-        try (Connection con = DatabaseExtension.src.getConnection()) {
-            for (final String sql : Arrays.asList(
+        try (Connection con = this.src.getConnection()) {
```

Comment:
@olenagerasimova `con` should be final

Reply:
@victornoel 
`PMD: D:\EO\ghman\src\test\java\com\g4s8\ghman\data\DatabaseExtension.java[57-57]: Unnecessary modifier 'final' on resource specification 'con': resource specifications are implicitly final (UnnecessaryModifier)`